review/reviewer/navigate.md

```diff
@@ -26,54 +24,54 @@ However, we're actually going in the direction of removing the FooWidget system
-CL is too large for you to figure out which parts are the major parts, ask the
+the PR. Look at these major parts first. This helps give context to all of the
+smaller parts of the PR, and generally accelerates doing the code review. If the
+PR is too large for you to figure out which parts are the major parts, ask the
 developer what you should look at first, or ask them to
-[split up the CL into multiple CLs](../developer/small-cls.md).
+[split up the PR into multiple PRs](../developer/small-prs.md).
 
-If you see some major design problems with this part of the CL, you should send
+If you see some major design problems with this part of the PR, you should send
 those comments immediately, even if you don't have time to review the rest of
-the CL right now. In fact, reviewing the rest of the CL might be a waste of
+the PR right now. In fact, reviewing the rest of the PR might be a waste of
 time, because if the design problems are significant enough, a lot of the other
 code under review is going to disappear and not matter anyway.
 
 There are two major reasons it's so important to send these major design
 comments out immediately:
 
--   Developers often mail a CL and then immediately start new work based on that
-    CL while they wait for review. If there are major design problems in the CL
-    you're reviewing, they're also going to have to re-work their later CL. You
+-   Developers often mail a PR and then immediately start new work based on that
+    PR while they wait for review. If there are major design problems in the PR
+    you're reviewing, they're also going to have to re-work their later PR. You
     want to catch them before they've done too much extra work on top of the
     problematic design.
 -   Major design changes take longer to do than small changes. Developers nearly
     all have deadlines; in order to make those deadlines and still have quality
     code in the codebase, the developer needs to start on any major re-work of
-    the CL as soon as possible.
+    the PR as soon as possible.
 
-## Step Three: Look through the rest of the CL in an appropriate sequence {#step_three}
+## Step Three: Look through the rest of the PR in an appropriate sequence {#step_three}
 
-Once you've confirmed there are no major design problems with the CL as a whole,
+Once you've confirmed there are no major design problems with the PR as a whole,
 try to figure out a logical sequence to look through the files while also making
 sure you don't miss reviewing any file. Usually after you've looked through the
 major files, it's simplest to just go through each file in the order that
 the code review tool presents them to you. Sometimes it's also helpful to read the tests
 first before you read the main code, because then you have an idea of what the
 change is supposed to be doing.
 
-Next: [Speed of Code Reviews](speed.md)
+Next: [Speed Of Code Reviews](speed.md)
```

Comment:
Are we just going for capitalization of all the words? Not sure what that's called. Whenever I write headlines I usually don't capitalize the minor words. I think that's called title case? I don't know. Just thought I'd ask about it. It's not a big deal either way.